cmd/locode/db/csvlocodedb/calls.go

```diff
@@ -0,0 +1,158 @@
+package csvlocodedb
+
+import (
+	_ "embed"
+	"encoding/csv"
+	"errors"
+	"os"
+	"sort"
+	"strconv"
+
+	locodedb "github.com/nspcc-dev/locode-db/cmd/locode/db"
+)
+
+var errRecordNotFound = errors.New("record not found")
+
+// Get returns a record by key. locodedb.Key is a struct that contains CountryCode and LocationCode.
+// locodedb.Record is a struct that contains CountryName, LocationName, SubDivName, SubDivCode, GeoPoint and Continent.
+func (db *DB) Get(key locodedb.Key) (rec *locodedb.Record, err error) {
+	rec = &locodedb.Record{}
+
+	locode := key.CountryCode().String() + key.LocationCode().String()
+
+	locodeCSV, found := locodesMap[locode]
+	if !found {
+		return nil, errRecordNotFound
+	}
+
+	country, countryFound := countriesMap[*key.CountryCode()]
+	if !countryFound {
+		return nil, errRecordNotFound
+	}
+	rec.SetCountryName(country)
+	rec.SetLocationName(locodeCSV.LocationName)
+	rec.SetSubDivName(locodeCSV.SubDivName)
+	rec.SetSubDivCode(locodeCSV.SubDivCode)
+	rec.SetGeoPoint(locodeCSV.Point)
+	rec.SetContinent(locodeCSV.Continent)
+
+	return
+}
+
+func (db *DB) Put(key locodedb.Key, rec locodedb.Record) error {
```

Comment:
Put shouldn't be per-line (there is no need for this), prepare all data in-mem beforehand and then sort/flush it.